src/themes/default/elements/segment.variables

```diff
@@ -29,7 +29,7 @@
 @groupedBoxShadow: @boxShadow;
 @groupedBorderRadius: @borderRadius;
 
-@nestedGroupMargin: @verticalMargin @verticalMargin;
+@nestedGroupMargin: @verticalMargin;
```

Comment:
vertical twice or a typo?

Reply:
From the git history back in 2015, it seems it was intention to have all margins the same value, but of course only one `@verticalMargin` is necessary to achieve the same result (i tested `@verticalMargin @horizontalMargin` in case it was a type, but this has a complete different visual (not looking right) result)